tests/utils/test_download.py

```diff
@@ -3,14 +3,19 @@
 
 import pytest
 
-from kale.utils.download import download_file_by_url
+from kale.utils.download import download_file_by_url, download_file_gdrive
 
 output_directory = Path().absolute().parent.joinpath("test_data/download")
 PARAM = [
     " https://github.com/pykale/data/raw/main/video_data/video_test_data/ADL/annotations/labels_train_test/adl_P_11_train.pkl;a.pkl;pkl",
     " https://github.com/pykale/data/raw/main/video_data/video_test_data.zip;video_test_data.zip;zip",
 ]
 
+GDRIVE_PARAM = [
+    "1U4D23R8u8MJX9KVKb92bZZX-tbpKWtga;demo_datasets.zip;zip",
+    "1SV7fmAnWj-6AU9X5BGOrvGMoh2Gu9Nih;dummy_data.csv;csv",
```

Comment:
Where are these located? Which Google account? Permanent?

Reply:
The first one was taken directly from the video loading example, and is in Raivo's Google account. The second one is in my account.

These are not permanent. Is there a PyKale google account? Maybe these dummy files can be added there.

Reply:
Thanks for the clarification. It is fine then. We do not have a PyKale Google account (yet). No need to create just for dummy files.